src/sonic-frr/patch/series

```diff
@@ -6,30 +6,12 @@
 0006-Link-local-scope-was-not-set-while-binding-socket-for-bgp-ipv6-link-local-neighbors.patch
```

Comment:
Can we make sure we change the patch number to align with the removed patches?

Reply:
I wanted to keep the old patches(which are no longer applicable) so that upgrading from the earlier FRR will be easy.

Reply:
As we have discussed in the meeting, we have deleted all the patches which are not needed now. Also, the ordering stays as is.